oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/TestUtils.java

```diff
@@ -93,6 +93,14 @@ public static DocumentNodeState asDocumentState(NodeState state){
         return null;
     }
 
+    public static void persistToBranch(NodeBuilder builder) {
+        if (builder instanceof DocumentRootBuilder) {
+            ((DocumentRootBuilder) builder).persist();
+            return;
+        }
+        fail("Not of type DocumentRootBuilder: " + builder.getClass().getName());
+    }
+
```

Comment:
Similar as above, can you please move this method between `resetRevisionClockToDefault()` and `disposeQuietly()`?